util-bukkit/src/main/i18n/templates/strings.properties

```diff
@@ -1045,415 +1033,385 @@ observer.tools.flyspeed.hyperspeed = Hyperspeed
 
 # {0} = a map name
 # {1} = an amount of seconds
-countdown.cycle.message = Cycling to {0} in {1}
-
+maps.cycle.message.verbose = Cycling to {0} in {1}
 # {0} = an amount of seconds
-countdown.cycle.message.no_map = Cycling in {0}
+maps.cycle.message.no_map = Cycling in {0}
 
 # {0} = a map name
-countdown.cycle.complete = Cycled to {0}
-countdown.cycle.complete.no_map = Cycled
+maps.cycle.complete.verbose = Cycled to {0}
+maps.cycle.complete.no_map = Cycled
 
 +           -----MATCHES-----
 + String belonging to functions that handle the different parts of a match
 + These functions include but is not limited to:
 + Teams, timelimits, winners/losers, objectives, joining(the match not the server)
 + NOTE: All strings belonging to a specific gamemode has their own category
 
-finalizedMatch.join.overMessage = Match is over
-finalizedMatch.join.cycleMessage = Please wait for the server to cycle
+match.join.denied.finished = Match is over
+match.join.denied.cycling = Please wait for the server to cycle
 
 # {0} = "1"
-start.needMorePlayers.ffa.singular = Waiting for {0} more player to join
-
+match.needMorePlayers.ffa.singular = Waiting for {0} more player to join
 # {0} = number of players that must join before the match can start
-start.needMorePlayers.ffa.plural = Waiting for {0} more players to join
-
+match.needMorePlayers.ffa.plural = Waiting for {0} more players to join
 # {0} = "1"
 # {1} = team name
-start.needMorePlayers.team.singular = Waiting for {0} more player to join {1}
-
+match.needMorePlayers.team.singular = Waiting for {0} more player to join {1}
 # {0} = number of players that must join before the match can start
 # {1} = team name
-start.needMorePlayers.team.plural = Waiting for {0} more players to join {1}
+match.needMorePlayers.team.plural = Waiting for {0} more players to join {1}
 
 # {0} = objective mode e.g. "Gold Core Mode"
 # {1} = time remaining until mode change
-match.objectiveMode.countdown = {0} in {1}
+match.objectives.modeCountdown = {0} in {1}
 
-match.destroyable.damageOwn = You may not damage your own objective.
-match.destroyable.repairOther = You may not repair an enemy objective.
-match.destroyable.repairDisabled = This objective may not be repaired.
-command.map.mapInfo.objective = Objective
+destroyable.damageOwn = You may not damage your own objective.
+destroyable.repairOther = You may not repair an enemy objective.
+destroyable.repairDisabled = This objective may not be repaired.
+maps.mapInfo.objective = Objective
 
 # Dont translate, keep as-is
-objective.credit.player.percentage = {0} ({1}%)
-
-objective.credit.etc = others
-objective.credit.many = many, many players
-objective.credit.unknown = unknown forces
+match.objective.credit.percentage = {0} ({1}%)
+match.objective.credit.etc = others
+match.objective.credit.many = many, many players
+match.objective.credit.unknown = unknown forces
+match.objective.willReceiveCredit = You will receive credit when this objective is completed.
```

Comment:
Should be grouped with `destroy` strings?

Reply:
`match.objective.modeCountdown` as well? Or is that string used for other events as well (like cos(tnt) diamond drops)

Reply:
Sure include that one